File: MT.Model/Dictionary/Level.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MT.Model
{
    /// <summary>
    /// Bảng cấp quản lý đề tài
    /// </summary>
    /// Create by: laipv:13.01.2018
    public class Level : EntityBase
    {
        [Key]
        public Int32 LevelID { get; set; }
        public string LevelCode { get; set; }
        public string LevelName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
    }
}

File: BECore/app/model/Level.js
/*
* Danh mục cấp quản lý đề tài
* Create by: laipv:13.01.2018
*/
Ext.define('QLDT.model.Level', {
    extend: 'Ext.data.Model',
    idProperty: 'LevelID',
    idType: 'Guid',
    
    code: 'LevelCode',
    name: 'LevelName',

    fields: [
        { name: "LevelID", type: "string" },
		{ name: "LevelCode", type: "string" },
		{ name: "LevelName", type: "string" },
		{ name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});
